src/pages/MyLibrary/LibraryBooks/OpenBook/OpenBook.js

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { Page, Document } from 'react-pdf';
+import StarRating from './StarRating';
+import useViewPort from './ViewPort/useViewPort';
+import { pdfjs } from 'react-pdf';
+pdfjs.GlobalWorkerOptions.workerSrc = `//cdnjs.cloudflare.com/ajax/libs/pdf.js/${pdfjs.version}/pdf.worker.js`;
+
+const OpenBook = props => {
+  const { book, setBook } = props;
+  const [numPages, setNumPages] = useState(null);
+  const [pageNumber, setPageNumber] = useState(1);
+  const { width, height } = useViewPort();
+
+  function onDocumentLoadSuccess({ numPages }) {
+    setNumPages(numPages);
+  }
+
+  const closeBook = () => {
+    setBook(book => !book);
+  };
+
+  const bookscale = () => {
+    let newwidth = width / 1920;
+    let newheight = height / 1019;
+
+    return newwidth >= newheight ? newheight : newwidth;
+  };
+
+  return (
+    <>
+      <Modal book={book} onClick={closeBook} />
+      <OpenBookTotal book={book}>
+        <DocumentBox>
+          <Document
+            file="PlantSiteManagementImprovementMethod.pdf"
+            onLoadSuccess={onDocumentLoadSuccess}
+          >
+            <Page pageNumber={pageNumber} scale={bookscale()} />
+          </Document>
+          <MoveButton
+            location="left"
+            onClick={() =>
+              pageNumber > 1 ? setPageNumber(pageNumber - 1) : null
+            }
+          />
+          <MoveButton
+            location="right"
+            onClick={() =>
+              pageNumber < numPages ? setPageNumber(pageNumber + 1) : null
+            }
+          />
```

Comment:
```suggestion
         <MoveButton
              location="left"
              onClick={() =>
                pageNumber > 1 && setPageNumber(pageNumber => pageNumber - 1)
              }
            />
            <MoveButton
              location="right"
              onClick={() =>
                pageNumber < numPages &&
                setPageNumber(pageNumber => pageNumber + 1)
              }
            />
```
조건의 결과가 한 가지만 있을 경우 삼항연산자가 아닌 && 연산자를 사용해주세요! 
또한 setState시 비동기적인 특성에 집중해서 이전의 상태값을 받아서 사용하는 방법을 사용해주세요~

Reply:
함수형으로 변경 했습니다.